### PR TITLE
fix(theme): fixed that all disabled elements used a disable background color instead of just disabled tabs

### PR DESCRIPTION
**Pull request by christoph-jerolimov**
## Hey, I just made a Pull Request!

TODO

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/redhat-developer/rhdh-plugins/blob/main/CONTRIBUTING.md#creating-changesets))
- [ ] Added or Updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

File: workspaces/theme/plugins/theme/src/utils/createComponents.ts [hunk 1]
```diff
@@ -436,7 +436,7 @@ export const createComponents = (themeConfig: ThemeConfig): Components => {
         root: {
           textTransform: 'none',
           minWidth: 'initial !important',
-          '&.Mui-disabled': {
+          '&:disabled': {
             backgroundColor: general.disabledBackground,
```
Comment (ciiay) [inline workspaces/theme/plugins/theme/src/utils/createComponents.ts:440]:
```suggestion
            backgroundColor: 'unset',
```
⬆️ fixes the following issue with tab button background color being gray.
image

**Review by ciiay — CHANGES_REQUESTED** [review]
Disabled buttons are fixed, just left a comment about the tab button being gray issue.

File: workspaces/theme/plugins/theme/src/utils/createComponents.ts (hunk 1 above)
Comment (christoph-jerolimov) [reply workspaces/theme/plugins/theme/src/utils/createComponents.ts:440]: Then I can just remove this disabled code. I thought setting a gray background to the tab was exactly what we want to make it more clear that the tab is disabled.

With just the different text Color is really hard to see if a tab is disabled. I will check with @ShiranHi and the PF 5/6 documentation.

Comment (christoph-jerolimov) [reply workspaces/theme/plugins/theme/src/utils/createComponents.ts:440]: @ciiay yeah this "idea" comes from PF. See: https://v5-archive.patternfly.org/components/tabs

image

**Review by ciiay — APPROVED** [review]
/lgtm

Thank you 🤝

File: workspaces/theme/plugins/theme/src/utils/createComponents.ts (hunk 1 above)
Comment (ShiranHi) [reply workspaces/theme/plugins/theme/src/utils/createComponents.ts:440]: Right, let's follow [PF guidelines](https://www.patternfly.org/components/tabs/design-guidelines#disabled-tabs). Do they use the same gray color for both light and dark mode?

Comment (christoph-jerolimov) [reply workspaces/theme/plugins/theme/src/utils/createComponents.ts:440]: @ShiranHi, I like that you asking the uncomfortable, on-point questions :rofl: 

Here we go: #129

**Pull request merged**